src/shared/telemetry/activation.ts

```diff
@@ -117,13 +126,13 @@ export async function handleTelemetryNoticeResponse(
             return
         }
 
-        const setting =
-            response === responseDisable ? SETTINGS_TELEMETRY_VALUE_DISABLE : SETTINGS_TELEMETRY_VALUE_ENABLE
-        getLogger().verbose(`Applying telemetry setting: ${setting}`)
-        await toolkitSettings.writeSetting<string>(AWS_TELEMETRY_KEY, setting, vscode.ConfigurationTarget.Global)
-
         setHasUserSeenTelemetryNotice(extensionContext)
-        ext.telemetry.notifyOptOutOptionMade()
+
+        // noticeResponseOk is a no-op
+
```

Comment:
Nit, I don't think we need whitespace here

Reply:
I initially had no newline there, but the comment looked out of place against the other response directly below it